src/wolfgame.py

```diff
@@ -1795,108 +1800,108 @@ def fday(cli, nick, chan, rest):
     if var.PHASE != "night":
         cli.notice(nick, messages["not_nighttime"])
     else:
-        transition_day(cli)
+        transition_day()
 
-# Specify force = "nick" to force nick to be lynched
-def chk_decision(cli, force="", end_game=True, deadlist=[]):
+# Specify force = user to force user to be lynched
+def chk_decision(force=None, end_game=True, deadlist=UserList()):
```

Comment:
The deadlist kwarg is a ticking time bomb waiting to happen (it was an issue back when the default was [] as well). Should default to None instead and figure out what to do in that case. Right now you create a new UserList that will never under any circumstance be cleared out should users get added to it (can users even be added to it? or do we make a copy right away and only operate on the copy?).

Reply:
Fair point, I'll look into it.